torch/csrc/jit/custom_operator.h

```diff
@@ -26,7 +26,10 @@ struct TORCH_API RegisterOperators {
   template <typename Implementation>
   C10_DEPRECATED_MESSAGE("torch::jit::RegisterOperators is deprecated. Please use torch::RegisterOperators instead.")
   RegisterOperators(const std::string& name, Implementation&& implementation) {
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wdeprecated-declarations"
```

Comment:
do you think it's possible to avoid these?

Reply:
Unfortunately, there doesn't seem to be a concept of "deprecated methods are allowed to call other deprecated methods", so whenever you call a deprecated method and don't want a warning, you have to add this - even if you're calling it from a method that's already deprecated.

This is ugly, I agree, but I don't have a better solution.

Reply:
the better approach is to remove all the deprecated invocations, right?  why are we even calling this method?

Reply:
Discussed offline and found a workaround. Thanks.